templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,168 @@
+    </division>
+{{end}}
+{{define "ticket-detail"}}
+    <table-cell indent="0">
+        <division margin="10 0 0 0">
+            <paragraph>
+                <text-chunk font="{{.FontName}}" font-size="{{.FontSize}}">{{.Name}}</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+{{end}}
+<table columns="2" margin = "0 0 10 10" column-widths="0.20 0.80">
+    <table-cell>
+        <image src="path('./templates/res/red-guitar.png')" width="100" height="160" margin="5 0 0 0"></image>
+    </table-cell>
+    <table-cell>
+        <division>
+            <division margin="0 0 0 15">
+            {{template "header" dict "FontSize" 19 "SubFontSize" 11 "Time" (formatTime .EventTime "02.01.2006 03:04PM")}}
+            </division>
+                <table columns="2" column-widths="0.3 0.7">
+                <table-cell>
+                    <table columns="1" margin="10 0 0 0">
+                        <table-cell>
+                            <division>
+                                <paragraph margin="0 0 0 10" line-height="1.1">
+                                    <text-chunk>E - ticket</text-chunk>
+                                </paragraph>
+                                <paragraph margin="2 0 0 10">
+                                    <text-chunk>{{.TicketNumber}}</text-chunk>
+                                </paragraph>
+                                <image src="qr-code" height="70" width="70" margin="5 0 0 10"></image>
+                            </division>
+                        </table-cell>
+                    </table>
+                </table-cell>
+                <table-cell>
+                    <table columns="2">
+                        {{range .Detail}}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldName }}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldValue }}
+                        {{end}}
+                    </table>
+                </table-cell>
+            </table>
+        </division>
+    </table-cell>
+</table>
+<table columns="2" margin="0 0 0 15">
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-left="0.5" vertical-align="top" indent="0.5" border-style="single">
+    <division margin="0 0 15 15">
+        <paragraph margin="10 0 10 0">
+            <text-chunk font="helvetica-bold" font-size="12">Rules of attendance</text-chunk>
+        </paragraph>
+        {{range .RulesOfAttendance}}
+            <paragraph line-height="1.3">
+                <text-chunk font-size="9">{{.}}</text-chunk>
+            </paragraph>
+        {{end}}
+    </division>
+ </table-cell>
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-right="0.5" indent="0" border-style="single">
+    <division margin="0 0 15 15">
+        <paragraph text-align="left" margin="10 0 10 0">
```

Comment:
Could use the shorthand form here for the margin attribute: 10 0 (top/bottom 10, left/right 0).
Also, please remove the `text-align` attribute. `left` is the default value already.